src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,147 @@
+package duke;
```

Comment:
Although it's not required, I feel that dividing the classes into multiple packages would be cleaner. 
E.g. `duke.storage`, `duke.task`, `duke.ui`, `duke.command`